docs/setup.mdx

```diff
@@ -102,12 +140,19 @@ follow the instructions for **your platform** below:
 
 ## Update
 
-The Oasis CLI includes a built-in command to upgrade to the latest version.
-To update, run `oasis update`.
+The Oasis CLI includes a built-in command to upgrade to the latest version
+since v0.15.0. To update, run `oasis update`.
 
 This will check for a newer version on GitHub, show you the release notes,
```

Comment:
The version is redundant IMO. But I would mention that this method is suitable for manual installations only.

```suggestion
If you installed Oasis CLI manually, the application includes a built-in
`oasis update` command which upgrades software to the latest version.

This command will check for a newer version on GitHub, show you the release notes,
```